### ISSUE TITLE
Create a new release for the SAF support

### ISSUE THREAD

**Issue by tareksander** [termux/termux-api-package#150]
SAF support has been merged here and in Termux:API and I noticed today that it isn't available in the package from the main repo.  
Could you make a new release for the package so SAF support is usable?

**Comment by agnostic-apollo** [comment]
Yeah, will do it. Already committed some stuff for Termux:API today, some fixes remain, will try doing them today/tomorrow.

**Comment by Efreak** [comment]
Not just SAF, either. `termux-notification-channel` is also unavailable.

**Comment by wschlich** [comment]
@Grimler91 please, when can you create a new release with SAF support scripts as suggested by @tareksander ?

**Comment by Benjamin-Loison** [comment]
If I'm not mistaken this issue now deserves being closed.
